app/src/main/java/org/schabi/newpipe/util/Utils.java

```diff
@@ -0,0 +1,11 @@
+package org.schabi.newpipe.util;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+public class Utils {
```

Comment:
Do we really need a separate class just for one function that rounds a number? I'd personally put it as a private function in Localization.. Otherwise this PR looks good to me. @TobiGr: What do you think?

Reply:
I agree

Reply:
Well, the point was it's about Maths, not Localization. I didn't want to create a MathUtils just for one function, so I created a general Utils file. But ok, I done that change.